src/app/components/ContractVerificationIcon/index.tsx

```diff
@@ -49,13 +55,13 @@ export const VerificationIcon: FC<{
   }
   const Component = noLink ? Box : (Link as React.ElementType)
   const componentProps = noLink ? {} : sourcifyLinkProps
-
+  const hideLabelMobile = hideLabel && isMobile
```

Comment:
I'll fix: table component should decide to hide label on mobile. Not ContractStatus, not VerificationIcon